client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/navigator-item-group/style.scss

```diff
@@ -1,12 +1,19 @@
 @import "@automattic/typography/styles/fonts";
 
 .pattern-assembler {
-	.pattern-layout__navigator-item-group {
+	.navigator-item-group:only-child {
+		.navigator-item-group__title {
+			display: none;
```

Comment:
This might better be removed after Colors/Fonts has been released (because I don't know if there's any case Colors/Fonts doesn't exist), but I see it won't be a problem 👍